docs/rig-ops-guide.md

```diff
@@ -60,6 +60,7 @@ Variable&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 `NODE_COOKIE` | Erlang cookie used in distributed mode, so nodes in cluster can communicate between each other. | nil
 `NODE_HOST` | Erlang hostname for given node, used to build Erlang long-name `rig@NODE_HOST`. This value is used by Erlang's distributed mode, so nodes can see each other. | nil
 `PROXY_CONFIG_FILE` | Configuration JSON file with initial API definition for API Proxy. Use this variable to pass either a path to a JSON file, or the JSON string itself. A path can be given in absolute or in relative form (e.g., `proxy/your_json_file.json`). If given in relative form, the working directory is one of RIG's `priv` dirs (e.g., `/opt/sites/rig/lib/rig_inbound_gateway-2.0.2/priv/` in a Docker container). | nil
+`PROXY_HTTP_RESPONSE_TIMEOUT` | In case an endpoint has `target` set to `http` and `response_from` set to `http_internal`, this is the maximum delay between an HTTP request and the corresponding HTTP response message. | 5000
```

Comment:
maybe it's me but it doesn't come accross that it's not the _synchronous_ http response you're talking about here